.github/workflows/cicd.yml

```diff
@@ -152,6 +152,58 @@ jobs:
       code_coverage: false
       gather_dumps: true
 
+  bpf2c_fuzzer:
```

Comment:
Isn't this in another PR as well? I am making changes there as well to capture generated artifacts on fuzzing crashes.

Reply:
yes, will remove from here